packs/st2/actions/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: "send_anonymous_install_data"
```

Comment:
start with `callhome` and when, when needed, if neededd, you can go `callhome_install` `callhome_pack` etc. My 0.02. I am very picky with the names FWIW.